app/_hub/kong-inc/opentelemetry/_index.md

```diff
@@ -34,7 +34,7 @@ params:
       required: false
       datatype: map
       value_in_examples:
-        - X-Auth-Token:secret-token
+        X-Auth-Token: secret-token
```

Comment:
@lena-larionova Do you know why this isn't rendering correctly in the output preview doc? https://deploy-preview-5215--kongdocs.netlify.app/hub/kong-inc/opentelemetry/3.0.x.html

Is it reading "X-Auth-Token" like it would the "required" or "datatype" fields?

Reply:
No, I wasn't aware ! Thank you to have pointed that out to me. First time contributing, I'm sorry :)

However, it seems the tool used to generate the doc examples doesn't handle correctly a map type parameter and a YAML dictionary as a value. It's probably why the current OpenTelemetry doc have the syntax error I'm trying the fix.

I just tried the YAML flow style syntax for the value, but it doesn't work either.

Do you know if that tool is Kong-made and if we can modify it ?

Reply:
> Do you know if that tool is Kong-made and if we can modify it ?

It sure is :) Let me take a look.